src/AddTodoForm.js

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
-let AddTodoForm = (props) => {
+
+
+
+let AddTodoForm = ({ onAddTodo }) => {
+    const [todoTitle, setTodoTitle] = React.useState([]);
+    
+    const handleTitleChange = (event) => {
+        const newTodoTitle = event.target.value;
+        setTodoTitle(newTodoTitle);
+    }
     const handleAddTodo = (event) => {
         event.preventDefault()
-        const todoTitle = event.target.elements.title.value;
+        
         console.log(`${todoTitle}`);
+        setTodoTitle("");
```

Comment:
I would suggest moving `setTodoTitle("");` at the end of the `handleAddTodo` function